asyncapi/hubspot/Package.md

```diff
@@ -0,0 +1,18 @@
+Listen to [Hubspot event API](https://developers.hubspot.com/docs/api/webhooks/) from Ballerina
```

Comment:
```suggestion
Listen to [HubSpot event API](https://developers.hubspot.com/docs/api/webhooks/) from Ballerina.
```

Reply:
@LahiruRajapaksha Shall I change as suggested? or do we have a convention on this wording? I mean in all triggers, we have used like `Listen to`

Reply:
@Drifftr requesting your feedback on this

Reply:
Changed as discussed with @RukshiW